cypress/support/helpers/getPageUrls.js

```diff
@@ -0,0 +1,18 @@
+const path = require('ramda/src/path');
+
+global.Cypress = { env: () => {} }; // Fake Cypress.env
+const services = require('../config/services');
+
+const getPageTypes = service => path([service, 'pageTypes'], services);
+
+const getPageUrls = ({ pageType, environment, isSmoke }) =>
```

Comment:
Do we want to default `isSmoke = false`, so that this can be reused elsewhere?

Reply:
isSmoke is defaulted to true, the idea being we don't want to run all tests by default